### ISSUE TITLE
Duplicated plist files

### ISSUE THREAD

**Issue by otaviocc** [leavez/cocoapods-binary#119]
Hi there. I was investigating a way to improve the build times of the project I'm currently working on and learned about this plugin.

But I'm facing a problem when using it. Running `bundle exec pod install` without the plugin several `.plist` files are created and added to all the projects in the workspace. No problem here, this is how CocoaPods works.

When I enabled the plugin and ran `bundle exec pod install`, the same files were created and added to the projects in the workspace. But, and here is the problem, another set o `.plist` files were added to the projects in the workspace in addition to the original ones. These new this files, created by the plugin, point to `.plist` files in `./Pods/_Precompiled/...` and these files are not there. Additionally, Xcode tries to copy these duplicated files (in a build phase added by CocoaPods ) and reports there are two sets of `.plist` files to the same thing.

In other words

1. The plugin adds plist files to the projects that are not in the disk
1. If these files were in the disk, Xcode would complain about the duplication on the build phase (like it does now, even without the files there).

Should the plugin create the .plist files and add them to all the projects? Why not use the ones CocoaPods already generates? If if does need to create new files, why does it add the ones gerenated by CocoaPods?

Thanks.

**Comment by otaviocc** [comment]
Closing this for inactivity.
